dnsserv_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	. "gopkg.in/check.v1"
+	"net"
+	"testing"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type DNSNameserverSuite struct{}
+
+var _ = Suite(&DNSNameserverSuite{})
+
+func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
Also, it looks as though we are testing how our UDPAddr complies with an address parsable by net.SplitHostPort. 

Instead of using net.SplitHostPort, you can also get a similar effect by making the following asserts on "output":

```
c. Assert(output.IP.String(), Equals, "127.0.0.1")
c. Assert(output.Port, Equals, 53)
```